packages/zosfiles/src/methods/upload/Upload.ts

```diff
@@ -854,10 +854,13 @@ export class Upload {
         };
     }
 
-    public static async uploadFile(session: AbstractSession, localPath: string, ussPath: string,
-        options: IUploadOptions): Promise<IZosFilesResponse> {
-        const tempOptions: Partial<IUploadOptions> = {};
-
+    public static async uploadFile(
+        session: AbstractSession,
+        localPath: string,
+        ussPath: string,
+        options: IUploadOptions
+    ): Promise<IZosFilesResponse> {
+        const tempOptions: Partial<IUploadOptions> = { ...options };
```

Comment:
I noticed that before this change, we only assigned `binary, encoding, localEncoding` to `tempOptions` within this function. With this change, all of the options are inherited from `options`, but then the logic re-assigns those variables before being passed to `fileToUssFile`. 

I don't believe this is a big deal, but is this change intentional? Technically, this will pass in more options to `fileToUssFile` than what was being passed in before.